alpha/apps/kaltura/modules/kmc/lib/kmcUtils.class.php

```diff
@@ -114,10 +114,10 @@ public static function getPartnersUiconfs($partner_id, $tag)
 		return $confs;
 	}
 
-	public static function getLiveAUiconf()
+	public static function getLiveUiconf($uiConfTag)
```

Comment:
I would change the name from getLiveUiconf to getLiveUiconfByTag

Reply:
ok